exchange/rpc.go

```diff
@@ -134,6 +119,7 @@ type Device struct {
 	Token              string             `json:"token"`
 	Name               string             `json:"name"`
 	Owner              string             `json:"owner"`
+	NodeType           string             `json:"nodeType"`
```

Comment:
I think we need a GetNodeType() on this object so that it can return device when the device type is set to device and when it's not set at all. We will have to support nodes where the nodeType is not set due to migration from 4.0 to 4.1.

Reply:
We should leave whatever the exchange returns to us. This way, we can decide if the field is defined or not in the code to make decisions.

Reply:
The exchange already returns "device" when the nodeType is empty.